src/main/include/subsystems/Climber.hpp

```diff
@@ -3,6 +3,10 @@
 #pragma once
 
 #include <frc/Solenoid.h>
+#include <networktables/NetworkTable.h>
+#include <networktables/NetworkTableEntry.h>
+#include <networktables/NetworkTableInstance.h>
```

Comment:
Put the other two includes in Climber.cpp, because that's where they're used.
```suggestion
#include <networktables/NetworkTableEntry.h>
```